Remove duplicate text from README.adoc of descriptor_indexing

burak-efe opened the pull request:
## Description
remove duplicate text

## General Checklist:

Please ensure the following points are checked:

- [x] My code follows the [coding style](https://github.com/KhronosGroup/Vulkan-Samples/tree/main/CONTRIBUTING.adoc#Code-Style)
- [x] I have reviewed file [licenses](https://github.com/KhronosGroup/Vulkan-Samples/tree/main/CONTRIBUTING.adoc#Copyright-Notice-and-License-Template)
- [x] I have commented any added functions (in line with Doxygen)
- [x] I have commented any code that could be hard to understand
- [x] My changes do not add any new compiler warnings
- [x] My changes do not add any new validation layer errors or warnings
- [x] I have used existing framework/helper functions where possible
- [x] My changes do not add any regressions
- [x] I have tested every sample to ensure everything runs correctly
- [x] This PR describes the scope and expected impact of the changes I am making

 Note: The Samples CI runs a number of checks including:
 - [x] I have updated the header Copyright to reflect the current year (CI build will fail if Copyright is out of date)
 - [x] My changes build on Windows, Linux, macOS and Android. Otherwise I have [documented any exceptions](https://github.com/KhronosGroup/Vulkan-Samples/tree/main/CONTRIBUTING.adoc#General-Requirements)

 If this PR contains framework changes:
 - [ ] I did a full batch run using the `batch` command line argument to make sure all samples still work properly

burak-efe committed:
Remove duplicate text from README.adoc of descriptor_indexing

burak-efe committed:
Update copyright year

burak-efe committed:
add jetbrains user IDE config folder to .gittignore

similar to .vs or .vscode

burak-efe committed:
Include the path parameter on file system error messages

asuessenbach commented:
I think, you should mention the additional changes in `.gitignore` and `std_filesystem.cpp` in the Description section above. Or maybe just move them into a separate PR.

SaschaWillems commented:
I second that. When I reviewed, it only had the documentation fix (which is fine). But a recent commit also changed code. Can you please revert that?

burak-efe committed:
Revert "Include the path parameter on file system error messages"

This reverts commit 12e23d20688a73f843ba913560ea48155268a5e9.

burak-efe committed:
Revert "add jetbrains user IDE config folder to .gittignore"

This reverts commit 65865fbfb569105c2fadc1bc3a88a77656572fb5.

burak-efe commented:
> I second that. When I reviewed, it only had the documentation fix (which is fine). But a recent commit also changed code. Can you please revert that?

Oh my bad, I was planning these for another request, I reverted that. It should be good now, sorry about that.
